src/node_platform.cc

```diff
@@ -407,10 +407,8 @@ void PerIsolatePlatformData::RunForegroundTask(std::unique_ptr<Task> task) {
     v8::HandleScope scope(isolate_);
     InternalCallbackScope cb_scope(env, Object::New(isolate_), { 0, 0 },
                                    InternalCallbackScope::kNoFlags);
-    task->Run();
-  } else {
-    task->Run();
```

Comment:
@juanarbol add a comment here that this else block is necessary and should not be removed.